README.md

```diff
@@ -56,99 +58,79 @@ $ pod install
 ```xml
-    <key>CFBundleURLTypes</key>
+<key>CFBundleURLTypes</key>
+<array>
+  <dict>
+    <key>CFBundleURLSchemes</key>
     <array>
-        <dict>
-            <key>CFBundleTypeRole</key>
-            <string>Editor</string>
-            <key>CFBundleURLName</key>
-            <string>$(PRODUCT_BUNDLE_IDENTIFIER)</string>
-            <key>CFBundleURLSchemes</key>
-            <array>
-                <string>msauth.<app.bundle.id></string>
-            </array>
-        </dict>
+      <string>msauth.com.microsoft.identitysample.MSALiOS</string>
     </array>
+  </dict>
+</array>
 ```
 
 2. Configure your application defaults
 
-In the `ViewControler.swift` file, update the `kClientID` variable with your client ID.
+In the `ViewControler.swift` file, update the `kClientID` variable with your Application (client) ID.
 
 ```swift
-    // Update the below to your client ID you received in the portal. The below is for running the demo only
+    // Update the below with the client ID you received in the portal. 
     
-    let kClientID = "<your-client-id-here>"
+    let kClientID = "66855f8a-60cd-445e-a9bb-8cd8eadbd3fa"
 ```
 
 ## Step 3: Run the sample
 
-Click the Run Button in the top menu or go to Product from the menu tab and select Run.
+1. Click the Run Button in the top menu or go to Product from the menu tab and click Run.
+2. Once the sample app launches, click on the 'Call Microsoft Graph API' button to go through the sign in flow and see the results from Microsoft Graph.
 
 ## Feedback, Community Help, and Support
 
-We use [Stack Overflow](http://stackoverflow.com/questions/tagged/msal) with the community to 
-provide support. We highly recommend you ask your questions on Stack Overflow first and browse 
-existing issues to see if someone has asked your question before. 
+We use [Stack Overflow](http://stackoverflow.com/questions/tagged/msal) with the community to provide support. We highly recommend you ask your questions on Stack Overflow first and browse existing issues to see if someone has asked your question before. 
 
-If you find and bug or have a feature request, please raise the issue 
-on [GitHub Issues](../../issues). 
+If you find a bug or have a feature request, please raise the issue on [GitHub Issues](../../issues). 
 
-To provide a recommendation, visit 
-our [User Voice page](https://feedback.azure.com/forums/169401-azure-active-directory).
+To provide a recommendation, visit our [User Voice page](https://feedback.azure.com/forums/169401-azure-active-directory).
 
 ## Contribute
 
-We enthusiastically welcome contributions and feedback. You can clone the repo and start 
-contributing now. Read our [Contribution Guide](Contributing.md) for more information.
+We enthusiastically welcome contributions and feedback. You can clone the repo and start contributing now. Read our [Contribution Guide](Contributing.md) for more information.
```

Comment:
Do we have contribution guide? I think we didn't have it for MSAL main repository so we removed reference to it from the readme.

Reply:
Good catch removed.